content/pairing.md

```diff
@@ -204,34 +204,26 @@ to include the Jupytext extension.
 
 You can pair the two formats in the classic Jupyter, Jupyter Lab,
 or the command line:
-<ul>
-<details>
-    <summary>
-        <b>1. Classic Jupyter Jupytext pairing</b>.
-    </summary>
-    <img src="../_static/01-classic.gif" width=80% height=80%>
-</details>
-    
-<details>
-    <summary>
-        <b>2. Jupyter Lab Jupytext pairing</b>
-    </summary>
-	<img src="../_static/02-jupyterlab.gif" width=80% height=80%>
-</details>
-
-<details>
-    <summary>
-		<b>3. Command line Jupytext pairing</b>
-    </summary>
-  <pre><code>
-  jupytext --set-formats ipynb,myst notebook.ipynb
-  </pre></code>
-  Then, update either the MyST markdown or notebook file:
-  <pre><code>
-  jupytext --sync notebook.ipynb
-  </pre></code>
-</details>
-</ul>
+
+**1. Classic Jupyter Jupytext pairing**
+
+![Animation showing pairing with Jupyter classic](_static/01-classic.gif)
+
+**2. JupyterLab Jupytext pairing**
+
+![Animation showing pairing with JupyterLab](_static/02-jupyterlab.gif)
+
+**3. Command line Jupytext pairing**
+
+```sh
+jupytext --set-formats ipynb,myst notebook.ipynb
+```
+
+Then, update either the MyST markdown or notebook file:
+
+```sh
+jupytext --sync notebook.ipynb
+```
```

Comment:
Rather than rely on raw html, you can get similar behavior using toggle-able elements from MyST, for example:

`````suggestion

```{admonition} **1. Classic Jupyter Jupytext pairing**
:class: toggle

![Animation showing pairing with Jupyter classic](_static/01-classic.gif)
```

```{admonition} **2. JupyterLab Jupytext pairing**
:class: toggle

![Animation showing pairing with JupyterLab](_static/02-jupyterlab.gif)
```

````{admonition} **3. Command line Jupytext pairing**
:class: toggle

```sh
jupytext --set-formats ipynb,myst notebook.ipynb
```

Then, update either the MyST markdown or notebook file:

```sh
jupytext --sync notebook.ipynb
```
````
`````

This would give similar (but appropriately-themed) collapsible elements similar to the original intent. The caveat is that these elements will not render properly in `jupyter notebook`. I think generally we recommend against using myst-specific functionality, but IMO it wouldn't hurt in this case as this is sort of a meta-tutorial that makes a lot more sense as a web page than as an executable notebook.

Reply:
This looks like an appropriate solution for the html, I'm still bummed about not having it show properly as a notebook but you're right - that is secondary for this document. So let's leave it like this and we can fix once we find an appropriate solution.